scripts/mina-local-network/split_precomputed_log.sh

```diff
@@ -9,6 +9,8 @@ fi
 
 ARCHIVE_URI=${ARCHIVE_URI:-postgres://postgres@localhost:5432/archive}
 PRECOMPUTED_LOG_FILE=$1
+OUTPUT_FOLDER=${2:precomputed_blocks}
```

Comment:
Why is it needed?

Reply:
this scripts generates a lot of files. So we want to enter to some dir first... It could be done differently without changing user directory though. This pr is porting one, so i can fix it in compatible branch